core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -798,43 +800,13 @@ public static Message asMessage(Object object) {
 		if (object instanceof Path) {
 			return new PathMessage((Path) object);
 		}
-		return new Message(null, object);
-	}
-
-	@Deprecated
-	public static Object asObject(Object object) {
-		if (object instanceof Message) {
-			return ((Message) object).asObject();
+		if (object instanceof MessageWrapper) {
+			return ((MessageWrapper<?>)object).getMessage();
```

Comment:
Ik zou in een MessageWrapper een Message verwachten maar niet andersom?

Reply:
Dat staat er toch ook? Parameter `object` hier is de parameter naar de static constructor-method, niet de het interne veld `object` van de `Message` class.